src/anet.c

```diff
@@ -491,10 +492,22 @@ int anetUnixServer(char *err, char *path, mode_t perm, int backlog)
     return s;
 }
 
+/* Accept a connection and also make sure the socket is non-blocking, and CLOEXEC.
+ * returns the new socket FD, or -1 on error. */
 static int anetGenericAccept(char *err, int s, struct sockaddr *sa, socklen_t *len) {
```

Comment:
The cluster mode connection establishment also calls through here, so need to get updated. https://github.com/redis/redis/blob/6476c8e856f790bef24eacd62c10c2df54e33f14/src/cluster.c#L719

Reply:
thanks. done.